### ISSUE TITLE
Problems with changing code after adding breakpoint

### ISSUE THREAD

**Issue by DamienCassou** [NicolasPetton/Indium#82]
1. add a breakpoint somewhere
2. `M-x indium-reload`
3. it should stop at the breakpoint. Continue execution
4. Go to the beginning of the line and press `C-o`
5. The breakpoint is now at the new line but I expected it to stay at its line (*first problem*)
6. Press `C-c b k` on the new line
7. The breakpoint is removed from the new line (good) but a new one is added to the line after it (*second problem*)

This process can only be reproduced once. As soon as you try again, the breakpoints reacts as expected.

**Comment by NicolasPetton** [comment]
I can reproduce the issue.

However, the first problem is not a bug, it's a feature (and it follows the behavior of the Chrome devtools). Why is it a problem for you?

**Comment by DamienCassou** [comment]
Step `5` was not clear enough, let me rephrase it. If I add a comment line on step `4`, the breakpoint is  at the comment line on step `5`. This is not what I expect: if I put a breakpoint on a `if` statement, I want the breakpoint to stay there, even if I add lines above (especially comment lines).

**Comment by NicolasPetton** [comment]
@DamienCassou can you confirm that the fix works fine for you as well?

**Comment by DamienCassou** [comment]
I'm not sure the bug is fixed. I did exactly the same again: the red dot is correctly placed, but the debugger stops one line after that.

**Comment by NicolasPetton** [comment]
That would be a different issue.

Could you open a ticket showing how to reproduce the problem?
